beacon-chain/core/altair/block.go

```diff
@@ -47,11 +47,11 @@ import (
 func ProcessSyncAggregate(ctx context.Context, s state.BeaconState, sync *ethpb.SyncAggregate) (state.BeaconState, error) {
 	votedKeys, votedIndices, didntVoteIndices, err := FilterSyncCommitteeVotes(s, sync)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "could not filter sync committee votes")
```

Comment:
how is this related to the PR? I don't mind the changes, I'm just curious

Reply:
ah I see, you made it better because you had to work through `GenerateFullBlockAltair` for tests

Reply:
Yeah, things were failing and I didn't know where